packages/mgt/README.md

```diff
@@ -48,48 +48,55 @@ The components are also available as [React components](https://learn.microsoft.
 
 [Watch the Getting Started Video](https://www.youtube.com/watch?v=oZCGb2MMxa0)
 
-You can use the components by referencing the loader directly (via unpkg), or installing the npm package
 
-### Use via mgt-loader:
+You can use the components by installing the npm packages or referencing the loader directly (via unpkg).
 
-```html
-<script src="https://unpkg.com/@microsoft/mgt@3/dist/bundle/mgt-loader.js"></script>
-```
+### Use via NPM:
 
-> NOTE: This link will load the highest available version of @microsoft/mgt in the range `>= 3.0.0 < 4.0.0`, omitting the `@3` fragment from the url results in loading the latest version. This could result in loading a new major version and breaking the application.
+The benefits of using MGT through NPM is that you have full control of the bundling process and you can bundle only the code you need for your site. First, add the npm package:
+
+```bash
+npm install @microsoft/mgt-components
+npm install @microsoft/mgt-msal2-provider
+```
 
-You can then start using the components in your html page. Here is a full working example with the Msal2 provider:
+Now you can reference all components and providers at the page you are using:
 
 ```html
-<script src="https://unpkg.com/@microsoft/mgt@3/dist/bundle/mgt-loader.js"></script>
-
-<mgt-msal2-provider client-id="[CLIENT-ID]"></mgt-msal2-provider>
+<script type="module">
+  import { Providers } from 'node_modules/@microsoft/mgt-element/dist/es6/index.js';
+  import { Msal2Provider } from 'node_modules/@microsoft/mgt-msal2-provider/dist/es6/index.js';
+  import { registerMgtLoginComponent, registerMgtAgendaComponent } from 'node_modules/@microsoft/mgt-components/dist/es6/index.js';
```

Comment:
Should we reference the `registerMgtComponents`  instead? This might be easier to "get started"? In our docs we'd be having a deeper dive in "how" to improve bundle size by registerig only the components you need...

Reply:
I've been thinking... Maybe not. Let's use best practices to build the muscle memory. And let's introduce the registerMgtComponents in another doc. So no action required here.